README.md

```diff
@@ -2,31 +2,230 @@
 
 # DSP-TOOLS - DaSCH Service Platform Tools
 
-dsp-tools is a command line tool that helps you to interact with the DaSCH Service Platform API. Consult the full 
-documentation [here](https://docs.dasch.swiss/latest/DSP-TOOLS).
+<!--bookmark-start-for-mkdocs-include-markdown-plugin-->
+DSP-TOOLS is a command line tool that helps you to interact with the DaSCH Service Platform API. This document is 
+intended for developers who want to work with the code of DSP-TOOLS. 
 
+| Hint                                                                                                                                                       |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| If you are a normal user of DSP-TOOLS, consult the [end user documentation](https://docs.dasch.swiss/latest/DSP-TOOLS) instead of this technical document. |
```

Comment:
Could you reformulate this? "If you are a normal user" sounds a bit unfriendly to me and I wouldn't know what sort of user I am if I consulted this page for the first time. How about something like "This is a technical documentation, for the end user documentation please visit ...".

Reply:
Ok, it's reformulated